### DISCUSSION TITLE
Example Use Cases

### DISCUSSION THREAD

**Discussion by moo** [dabreegster/abstreet#818] [General]
I wish I'd seen this sooner (as the Green Lake Outer Loop Survey ends tomorrow Mon, Dec 6), but these are great possibilities to explore: 

Segment 2: W Green Lake Dr N
15. W Green Lake Dr N currently has one travel lane in each direction and parking on the north side of the street. To complete a full outer walking biking loop, we’ll need to use either a travel lane or parking lane on this street. Which street configuration option for W Green Lake Dr N makes most sense to you?

- Option 1: Remove street parking and maintain vehicle travel lanes in both directions, and add a two-way walking and biking path on the lake side of the street.
- Option 2: Maintain street parking, but remove either a northbound or southbound vehicle travel lane, and add a two-way walking and biking path on the lake side of the street.
- Option 3: Remove parking, two-way travel, bike lanes on each side

From ~ 5th page of survey here: https://www.surveymonkey.com/r/P7JTWJ2

**Comment by dabreegster** [comment]
Just quickly mocking up some of the options...

Option 1: http://play.abstreet.org/dev/abstreet.html?--dev&system/us/seattle/maps/north_seattle.bin&--edits=remote/1b99ef37b6a38c7319d4790bdaef088f&--cam=18.76/47.68192/-122.34307

Option 2, leaving the WB lane to merge onto Aurora northbound: http://play.abstreet.org/dev/abstreet.html?--dev&system/us/seattle/maps/north_seattle.bin&--edits=remote/208b73d0e9156b060eadc2bedd84efb3&--cam=18.76/47.68192/-122.34313

Option 3: http://play.abstreet.org/dev/abstreet.html?--dev&system/us/seattle/maps/north_seattle.bin&--edits=remote/c36985bac44ba2cf9349a0c5bfe9d589&--cam=20.33/47.68163/-122.34348

Option 3 seems weird, since the cycletrack is two-way starting at Green Lake Dr N. If you're going counter-clockwise around the loop, why switch sides for a few streets?

**Reply by dabreegster** [reply]
Also, if you adjust any of these edits, note you can click the proposal name and "share proposal" now -- that's how I generated those URLs.
![Screenshot from 2021-12-05 22-11-25](https://user-images.githubusercontent.com/1664407/144765933-b2239017-663e-4410-a739-08858d472020.png)

**Comment by moo** [comment]
Here's another good one: 
[A $0 fix for the notorious 9th and Mercer intersection](https://www.seattlebikeblog.com/2022/04/29/watch-a-0-fix-for-the-notorious-9th-and-mercer-intersection/) 
It's a signal timing change! So, I threw it in: 
http://play.abstreet.org/0.3.19/abstreet.html?--dev&system/us/seattle/maps/slu.bin&--edits=remote/7653963ed696606ec8415f3f56bb7f8a

I think I changed the timing to have people on the sidewalk, in the bike lane and traffic going straight north first. _Then_ the phase for southbound traffic to turn east to Mercer. 

It looks like turns at this intersection need to be adjusted in general, and the bike lane and crosswalk do render funny, but it's great to be able to visualize this so quickly!

**Reply by dabreegster** [reply]
Hey! Tom's idea is brilliant in its simplicity; I hope y'all're get traction on this easily, and they don't use SCOOT as some excuse why it's not so simple a fix.

The current version renders as a total mess:
![Screenshot from 2022-05-02 09-49-35](https://user-images.githubusercontent.com/1664407/166209111-58d25f6d-02e7-4692-8725-2b03d9c67fad.png)
Problem 1 was a duplicate extra cycletrack, removed in https://www.openstreetmap.org/way/984500536. Problem 2 is the traffic signal being split into two pieces. I just fixed the latter in https://www.openstreetmap.org/changeset/120450564, and the result looks way better:
![Screenshot from 2022-05-02 09-49-45](https://user-images.githubusercontent.com/1664407/166209228-5b0f02d7-c170-4031-a681-9259764208d2.png)
You should be able to express the current and proposed timing pattern now. I wouldn't hold out hope for getting the simulation to show something realistic -- maybe if we clipped the area just around here, got the travel demand model right, and disabled the "don't block the box" default...

You could use the desktop version and reimport the SLU area to try the changes immediately. I'm aiming to do a release tonight and include this updated Seattle OSM data; I'll ping here when it's ready. Super glad to see the "upload proposal" and share URL experience worked!

**Reply by dabreegster** [reply]
You'll have to make the edits again, but http://play.abstreet.org/0.3.20/abstreet.html?--dev&system/us/seattle/maps/slu.bin should be way more realistic.

Also I vaguely recall SNG talking about home zones and traffic calming diverters. In case you haven't seen it, http://ltn.abstreet.org has been my project for the last few months. Beyond some of those diagonal diverters (they have some fancy name I forgot) just west of Aurora, I don't think LTNs have much traction in Seattle, but it's a cool idea potentially. Go put up road closures and create the Pine/Pike superblock again. :)
